package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ainft-factory-discord-bot",
+  "version": "1.0.2",
+  "main": "index.js",
+  "repository": "git@bitbucket.org:comcomai/ainft-factory-discord-bot.git",
+  "author": "hyeonwoong <hyeonwoong@comcom.ai>",
+  "license": "MIT",
+  "engines": {
+    "node": ">=16.0.0"
```

Comment:
I think we've downgraded the required node version for ainft-js before:
https://github.com/ainft-team/ainft-js/blob/main/package.json#L7

What about setting 12.0.0 here as well?

Reply:
I also looked at other dependencies, [discord js v13 requires node 16.6 or higher](https://discordjs.guide/additional-info/changes-in-v13.html#before-you-start). I guess I need to fix it with 16.6.

Reply:
Updated! Thanks for checking this.

Reply:
What about adding some contents or removing tools/create_ainft_server_app/README.md?

Reply:
question) it looks like different cases are used in this project. e.g.
src/services/AinftFactory.ts - pascal case
src/types/ainft-factory.ts - kebab case
src/events/interactionCreate.ts - camel case
tools/create_ainft_server_app/ - underscores

just wondering if we have any rules for them or just allow any cases in this project?

Reply:
1. What I was going to do for my next PR went in by mistake. Deleted.
2. I couldn't check earlier because we were working fast. Most of the files use camelCase, so I changed the above files to camelCase.